lsp-bridge-ref.el

```diff
@@ -941,12 +967,24 @@ Function `move-to-column' can't handle mixed string of Chinese and English corre
   (setq lsp-bridge-ref-temp-visit-buffers nil)
   ;; Kill search buffer.
   (kill-buffer lsp-bridge-ref-buffer)
-  ;; Restore window configuration before search.
   (when lsp-bridge-ref-window-configuration-before-search
-    (set-window-configuration lsp-bridge-ref-window-configuration-before-search)
+    ;; Restore window configuration before search if we delete-other-window before search.
+    ;; Or if we are inserting current line and the original window was deleted, we need to restore the window.
+    (if (or lsp-bridge-ref-delete-other-windows
+            (and insert
+                 (not (window-valid-p lsp-bridge-ref-requst-search-window))))
+        (set-window-configuration lsp-bridge-ref-window-configuration-before-search)
+      ;; Other we just delete the lsp-bridge-ref window, and select the original window.
+      (progn
+        (delete-window)
+        (select-window lsp-bridge-ref-requst-search-window)))
     (goto-char lsp-bridge-ref-buffer-point-before-search)
     (setq lsp-bridge-ref-window-configuration-before-search nil)
-    (setq lsp-bridge-ref-buffer-point-before-search nil)))
+    (setq lsp-bridge-ref-buffer-point-before-search nil))
+  ;; clear saved window which requested search.
+  (if lsp-bridge-ref-open-file-in-requset-window
+      (setq lsp-bridge-ref-requst-search-window nil))
+  )
```

Comment:
If i understand correctly, lsp-bridge-ref-request-search-window should reset to nil always, don't need ```if``` check here.

Reply:
Yeah, you are right.